src/pages/BalanceNew/StuckOraib/useGetOraiBridgeBalances.tsx

```diff
@@ -1,18 +1,16 @@
+import { Coin, StargateClient } from "@cosmjs/stargate";
 import { TokenItemType, tokens } from "config/bridgeTokens";
 import {
-    ORAI_BRIDGE_CHAIN_ID,
-    ORAI_BRIDGE_LCD
+    ORAI_BRIDGE_CHAIN_ID, ORAI_BRIDGE_RPC
 } from 'config/constants';
 import { toDisplay } from 'libs/utils';
 import { useEffect, useState } from 'react';
-import token from "reducer/token";
-import axios from 'rest/request';
-import { Balance, BalancesOraiBridge } from '../type';
 
 export type RemainingOraibTokenItem = TokenItemType & { amount: string };
 const ORAIB_DECIMALS = 18;
```

Comment:
this part should be edited. ORAIB_DECIMALS is not always 18. You can get the actual decimals based on the token denom filtered from the list in bridgeTokens.ts file